packetbeat/tests/system/test_0009_pgsql.py

```diff
@@ -10,15 +10,15 @@ def test_select(self):
         )
         self.run_packetbeat(pcap="pgsql_request_response.pcap")
 
-        objs = self.read_output()
+        objs = self.read_output(required_fields=TRANS_ECS_REQUIRED_FIELDS)
```

Comment:
I only added this to one of the assertions for now. I plan to change `read_output()` to make it check for all these fields by default after every protocol is updated.